apps/prairielearn/src/lib/markdown.test.ts

```diff
@@ -2,162 +2,140 @@ import { assert } from 'chai';
 
 import * as markdown from './markdown.js';
 
-const testMarkdownQuestion = (question, expected) => {
+async function testMarkdownQuestion(question, expected) {
```

Comment:
Why was this changed to an async function?

Consider adding types while you're here.

Reply:
`testMarkdown` was async originally, as I converted it to a regular function (instead of const) I added async, and ended up adding async here too. Changed in f82e6cf49.